### ISSUE TITLE
net0 limit on games

### ISSUE THREAD

**Issue by Vodkanakas** [aldostools/webMAN-MOD#489]
Is there a limit to the amount of games that can be loaded via ps3netserv for webman mod. I got about 350 psx games all in bin/cue format yet only 321 show. If I have it scan for ps2 and psx games only 130 ps2 show of my 167 titles and no psx games show. Is this a setting or a memory limit?

**Comment by aldostools** [comment]
Yes there is a limit. It can be changed if you go to /setup.ps3 and change the profile for the memory usage.
If you choose Max PSX+ or if you set 3072KB [MC]: to "4-bg", the scanning should be able to read more games.

![image](https://user-images.githubusercontent.com/974897/107312758-85914d00-6a67-11eb-9464-d90d8a433c50.png)

**Comment by Vodkanakas** [comment]
Thank you.

**Comment by Joonie86** [comment]
No further feedback
